client/wildcard/src/components/Typography/Label/Label.tsx

```diff
@@ -25,16 +26,45 @@ export const Label = React.forwardRef((props, reference) => {
         isUnderline,
         isUppercase,
         className,
+        onClick,
         ...rest
     } = props
 
+    const mergedRef = useMergeRefs([reference])
+
+    // Listen all clicks on the label element in order to improve click-to-focus logic
+    // for contenteditable="true". By default, label element's native behavior (click to focus a first input element)
+    // doesn't work with content editable element.
+    // Since we use content editable inputs (codemirror search box input) and labels together in some
+    // consumers we need to support this behavior manually for content editable elements.
+    const handleClick = (event: MouseEvent<HTMLLabelElement>): void => {
+        const htmlForAttribute = mergedRef.current?.getAttribute('htmlFor')
+
+        if (htmlForAttribute) {
+            onClick?.(event)
+            return
+        }
+
+        // Extend labelable elements set with contenteditable elements
+        const labelableElement = mergedRef.current?.querySelector<HTMLElement>(
+            'input, keygen, meter, output, progress, select, textarea, [contenteditable="true"]'
```

Comment:
An empty string in HTML for a boolean attribute is equivalent to setting it to the string `true`
```suggestion
            'input, keygen, meter, output, progress, select, textarea, [contenteditable=""], [contenteditable="true"]'
```